pyuvsim/tests/test_antenna.py

```diff
@@ -47,7 +48,38 @@ def test_jones_set_spline(cst_beam, hera_loc):
     altaz[:, 0] = alts.flatten()
     altaz[:, 1] = azs.flatten()
 
-    antenna.get_beam_jones(array, altaz, 150e6, interpolation_function='az_za_simple')
+    jones_matrix = antenna.get_beam_jones(
+        array, altaz, 150e6, interpolation_function='az_za_simple'
+    )
+
+    # These are just values from a run, so this just tests for unexpected changes.
+    expected_jones = np.array(
+        [
+            [
+                [
+                    -4.57061296e-04 - 3.88626249e-04j,
+                    -7.28285993e-05 - 1.45479743e-04j,
+                    -7.28285993e-05 - 1.45479743e-04j
+                ], [
```

Comment:
is it just me, or would I have expected black to put these two '],' and '[' on different lines? How weird.

Reply:
We haven't blackified this repo yet (it's coming after the lunar sim branch is merged), so I did this by hand. It'll probably change when we use black.

Reply:
OH! that'll do it! carry on.